Added SPEC for rpmbuild

xvitaly opened the pull request:
Added -install section in default makefile (needed by RPM-build).

Added SPEC for rpmbuild. RPM packages for Fedora 20+, CentOS, RHEL and openSUSE can be built by `rpmbuild -ba purple-skypeweb.spec`.

EionRobb commented:
The BuildRequires sections aren't quite right.  The plugin doesn't use openssl (wouldn't be able due to licensing issue) but does require libjson-glib.

Also would be nice if it installed the skype theme file as well as the icons :)

xvitaly commented:
> The BuildRequires sections aren't quite right. The plugin doesn't use openssl (wouldn't be able due to licensing issue) but does require libjson-glib.

Fixed.

> Also would be nice if it installed the skype theme file as well as the icons :)

As separate package?

xvitaly commented:
> The BuildRequires sections aren't quite right. The plugin doesn't use openssl (wouldn't be able due to licensing issue) but does require libjson-glib.

Fixed.

> Also would be nice if it installed the skype theme file as well as the icons :)

Added.

EionRobb commented:
Sorry for taking so long to merge this
